### PR TITLE
Restructured pages and simplified home page

### PR DESCRIPTION
**Pull request by mssola**
First of all, I have upgraded this to the latest Jekyll. The version we
were using was *really* old, and it failed on Travis. Moreover, we are
now taking advantage of Jekyll's SASS support. This comes out of the
box since Jekyll 3.0, and it greatly simplifies development environment.

Moreover, I have dropped Gulp, which included *lots* of dependencies.
Instead, I've added a `package.json` file, which is then picked up by
Yarn. This results in a much lighter `node_modules` directory. Moreover,
I've also added webpack for the javascript. The removal of Gulp has also
involved a re-structure of the assets directories.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

**Comment by mssola** [comment]
@vitoravelino this is kind of huge 😆 I'll post a couple of screenshots of the new main page:

2018-01-30-114446_1916x1037_scrot

**Comment by mssola** [comment]
I'm now working on the Travis part, but feel free to review what's already in there 👍

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "env": {
+    "jquery": true,
+    "browser": true,
+    "es6": true
+  },
+  "extends": "airbnb-base",
+  "globals": {
```
Comment (vitoravelino) [inline .eslintrc]: No need of this.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "no-unused-vars": [2, {
+      "args": "all",
+      "argsIgnorePattern": "^_"
+    }],
+    "object-shorthand": 0,
+    "no-plusplus": 0,
+    "no-void": 0,
+    "import/extensions": ["error", "always", {
+      "js": "never",
+      "vue": "never"
```
Comment (vitoravelino) [inline .eslintrc]: Same here.

**Comment by vitoravelino** [comment]
Fonts changed. I saw that `portus.scss`  does not contain the same imports and order compared to the previous file. The one that will fix the font issue is the `_type` partial.

**Comment by vitoravelino** [comment]
screenshot-20180130102727-1337x999

**Comment by vitoravelino** [comment]
Dropdown was not working in smaller screens. Probably need to import the dropdown from bootstrap in the main js file.

**Comment by vitoravelino** [comment]
screenshot-20180130103450-810x172

**Comment by vitoravelino** [comment]
You don't need to `import $ from 'jquery'`. While I was trying to upgrade jquery I also realized that `import 'jquery'` in the main file is not needed when provider plugin is being used. Double check just to be sure.

**Comment by vitoravelino** [comment]
`manifest.json` to gitignore? It's not being used by anyone afaik.

**Comment by vitoravelino** [comment]
Add `.sass-cache` to gitignore.
Add `*.log` to gitignore.

File: .eslintrc (hunk 2 above)
Comment (mssola) [reply .eslintrc]: Done

File: .eslintrc (hunk 1 above)
Comment (mssola) [reply .eslintrc]: Done

**Comment by mssola** [comment]
> Dropdown was not working in smaller screens. Probably need to import the dropdown from bootstrap in the main js file.

Done

> You don't need to import $ from 'jquery'. While I was trying to upgrade jquery I also realized that import 'jquery' in the main file is not needed when provider plugin is being used. Double check just to be sure.

Done.

> manifest.json to gitignore? It's not being used by anyone afaik.

Do you mean the one on `javascripts/dist` ?

> Add .sass-cache to gitignore. Add *.log to gitignore.

Done.

**Comment by vitoravelino** [comment]
> Do you mean the one on javascripts/dist ?

Yes.

**Comment by vitoravelino** [comment]
I guess there still something broken in the css. Investigating...

screenshot-20180130120530-286x383
screenshot-20180130120318-245x367

**Comment by vitoravelino** [comment]
Replace `fade` with `darken` utility function.

**Pull request merged**